cranelift/isle/docs/language-reference.md

```diff
@@ -1508,7 +1508,7 @@ The grammar accepted by the parser is as follows:
             | "(" <ident> <pattern-arg>* ")"
 
 <pattern-arg> ::= <pattern>
-                | "<" <expr>  ;; in-argument to an extractor
```

Comment:
I could be very wrong about this. But having this kind of infix operator defined in S-expression feels weird and I couldn't find it in the extractor parser, so I think this is a typo?

Reply:
This is outdated syntax, sorry -- we removed argument polarity a while ago. The whole `<expr>` case needs to go away and you can inline `<pattern>` for `<pattern-arg>` since we don't have the distinction anymore.